src/main/flight/pid_luxfloat.c

```diff
@@ -189,7 +191,17 @@ void pidLuxFloat(const pidProfile_t *pidProfile, const controlRateConfig_t *cont
 #endif
                 if (FLIGHT_MODE(ANGLE_MODE)) {
                     // ANGLE mode
-                    angleRate = errorAngle * pidProfile->P8[PIDLEVEL] / 16.0f;
+                    const float current_attitude = attitude.raw[axis] - angleTrim->raw[axis];
+                    const float d_error = (current_attitude - lastAttitude[axis]) / getdT();
```

Comment:
I was thinking about that but then I realized it was being used elsewhere for division with no check. Specifically line 128